languages/en.ini

```diff
@@ -1457,7 +1459,7 @@ widen_prefix = "Try widening your search to"
 wiki_link = "Provided by Wikipedia"
 with filters = "with filters"
 Year of Publication = "Year of Publication"
-Yesterday = "Yesterday"
+;Yesterday = "Yesterday"
```

Comment:
I've uncommented this for now because it was causing the CI build to fail. We can decide whether or not to clean out the two redundant translations as part of the overall language file translation after the core mechanism is finalized.